pkgs/applications/emulators/bsnes/ares/default.nix

```diff
@@ -31,45 +34,58 @@ stdenv.mkDerivation (finalAttrs: {
   patches = [
     ./000-dont-rebuild-on-install.patch
     ./001-fix-ruby.patch
+    ./002-sips-to-png2icns.patch
```

Comment:
I read that sips has been included in sdk 10.3, so we should have it available in 11.0. Is it missing? If so, maybe it's an issue to raise.

Reply:
Yes. And the issue actually dates back to #102263.

Reply:
In the meantime, I guess sips comes with macOS itself, not with the SDK.